CHANGELOG.md

```diff
@@ -1,6 +1,30 @@
 # Changelog for the Mapbox Maps SDK for Android
 
-Mapbox welcomes participation and contributions from everyone.  If you'd like to do so please see the [`Contributing Guide`](https://github.com/mapbox/mapbox-gl-native/blob/master/CONTRIBUTING.md) first to get started.
+Mapbox welcomes participation and contributions from everyone. Please read [`Contributing Guide`](https://github.com/mapbox/mapbox-gl-native/blob/master/CONTRIBUTING.md) to get started.
+
+## 9.1.0
+This beta release includes a feature with a known issue: the `within` expression evaluates point features inconsistently across zoom levels if the point lies near the boundary of a GeoJSON object ([#16301](https://github.com/mapbox/mapbox-gl-native/issues/16301])).
+### Features
+ - Added `within` expression for testing whether an evaluated feature lies within a given GeoJSON object. `within` accepts `"Feature"`, `"FeatureCollection"`, `"Polygon"`, and `"MultiPolygon"` as valid GeoJSON objects. ([#198](https://github.com/mapbox/mapbox-gl-native-android/pull/198))
```

Comment:
If it’s likely for a developer to create one of these geometries using the `com.mapbox.geojson` package, then perhaps mention the runtime types instead of the GeoJSON type strings.

```suggestion
 - Added `within` expression for testing whether an evaluated feature lies within a given GeoJSON object. `within` accepts `Feature`, `FeatureCollection`, `Polygon`, and `MultiPolygon` as valid GeoJSON objects. ([#198](https://github.com/mapbox/mapbox-gl-native-android/pull/198))
```